extensions/wikia/Email/Controller/DiscussionController.class.php

```diff
@@ -80,6 +80,16 @@ private function getDiscussionsLink() {
 		return $this->wiki->city_url . 'd';
 	}
 
+	/**
+	 * Wrap the provided text in <nowiki> tags. This escapes the text preventing
+	 * it from being parsed as wikitext.
+	 * @param $text
+	 * @return string
+	 */
+	protected function wrapTextInNoWikiTags( $text ) {
+		return "<nowiki>" . $text . "</nowiki>";
```

Comment:
don't we use apostrophes for strings?

Reply:
Single quotes? Yea, does look like those are preferred: https://www.mediawiki.org/wiki/Manual:Coding_conventions/PHP#String_literals

I'll update.